dbms/src/Storages/Page/V3/Universal/UniversalPageIdFormatImpl.h

```diff
@@ -164,6 +165,13 @@ struct UniversalPageIdFormat
         return buff.releaseStr();
     }
 
+    // LOCAL_PREFIX type keyspace_id(64 bits)
```

Comment:
Why is it stored in local page storage?
If this page is later uploaded as a checkpoint, and then downloaded by another store, can this store decrypt the page correctly?

Reply:
Page in S3 is not encrypted.